launcher/src/components/UI/base-header/components/modals/UpdatePanel.vue

```diff
@@ -339,37 +339,23 @@ const searchOsUpdates = async () => {
   searchingForOsUpdates.value = true;
   setTimeout(async () => {
```

Comment:
is this `setTimeout` even needed?

Reply:
i guess if the function is "to fast" the user will have no response from the UI but 10 ms will be to fast as well ^^

Reply:
You're absolutely right, it's not needed but since it did exist, i just kept it there with a minimum timeout to let others decide what to do with it :)